WoWDatabaseEditor.Common/WDE.CMMySqlDatabase/Models/CreatureTemplate.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using LinqToDB.Mapping;
+using WDE.Common.Database;
+
+namespace WDE.CMMySqlDatabase.Models
+{
+    /// <summary>
+    /// Creature System
+    /// </summary>
+    [Table(Name = "creature_template")]
+    public class CreatureTemplateWoTLK : ICreatureTemplate
+    {
+        [Identity]
+		[Column("Entry"                 , IsPrimaryKey = true )] public uint    Entry                  { get; set; } // mediumint(8) unsigned
```

Comment:
Could you please mass replace tabs with four spaces for consistency?

Reply:
fixed